src/xtgeo/surface/regular_surface.py

```diff
@@ -1975,7 +1975,15 @@ def get_dataframe(
 
         return pd.DataFrame(entry)
 
-    dataframe = get_dataframe  # for compatibility backwards
+    def dataframe(self, **kwargs):
+        """Deprecated; see method get_dataframe()."""
+        warnings.warn(
+            "The dataframe() is deprecated and will be removed in xtgeo "
+            "version 5. Use get_dataframe() instead",
+            PendingDeprecationWarning,
+        )
+
+        return self.get_dataframe(**kwargs)
```

Comment:
Not really the place for this comment maybe, and not relevant to this PR in particular, but I think it will be a good idea to harmonize the values that `get_dataframe()` accepts before we make those changes. It would be nice if they were consistent

Reply:
OK, we can consider. Not sure if it is doable, though, as the surface data are different that XYZ data in many aspects